Gruntfile.js

```diff
@@ -0,0 +1,140 @@
+        return fs.statSync(modules_dir + file).isDirectory();
+    });
+
+     _.each(modules, function (name) {
+        projectTemplates[name] = {
+            cwd: 'modules/' + name,
+            src: '**/*.html',
+            dest: 'modules/templates.js'
+        };
+    });
+
+    if (grunt.option('help')) {
+        require('load-grunt-tasks')(grunt);
+    } else {
+        require('jit-grunt')(grunt, {
+            ngtemplates: 'grunt-angular-templates'
+        });
+    }
+
+    require('time-grunt')(grunt);
+
+    var concatConfig = {
+        target: {
+            src: [
+                'modules/**/**/*.js'
+            ],
+            dest: 'dist/angular-feincms-pages.js'
+        }
+    };
+
+    var uglifyConfig = {
+        target: {
+            files: {
+                'dist/angular-feincms-pages.min.js': 'dist/angular-feincms-pages.js'
+            }
+        }
+    };
+
+    grunt.initConfig({
+        // Configurable paths
+        config: {
+            modules: 'modules',
+            lib: 'bower_components',
+            tests: 'tests',
+            files: {
+                lint: [
+                    '<%= config.modules %>/**/scripts/**/*.js',
+                    '!<%= config.modules %>/**/templates.js',
+                    'Gruntfile.js'
+                ],
+                templatesJS: 'modules/**/templates.js',
+                templatesHTML: 'modules/**/*.html'
+            }
+        },
+        watch: {
+        templates: {
+                files: 'modules/**/*.html',
+                tasks: 'ngtemplates'
+            },
+            scripts: {
+                files: 'modules/**/scripts/**/*.js',
+                tasks: ['build', 'eslint']
+            }
+        },
+        eslint: {
+            all: {
+                options: {
+                    config: '.eslintrc'
+                },
+                src: '<%= config.files.lint %>'
+            }
+        },
+        jscs: {
+            options: {
+                config: '.jscsrc'
+            },
+            src: '<%= config.files.lint %>'
+        },
+        ngtemplates: _.extend({
+                options: {
+                    htmlmin: {
+                        collapseBooleanAttributes: true,
+                        collapseWhitespace: true,
+                        removeAttributeQuotes: true,
+                        removeComments: true,
+                        removeEmptyAttributes: true,
+                        removeScriptTypeAttributes: true,
+                        removeStyleLinkTypeAttributes: true
+                    }
+                }
+            },
+            projectTemplates
+        ),
+        concat: concatConfig,
+        uglify: uglifyConfig
+    });
+
+    grunt.registerTask('lint', [
+        'eslint',
+        'jscs'
+    ]);
+
+    grunt.registerTask('default', [
+        'lint'
+    ]);
+
+    grunt.registerTask('travis', 'Run the tests in Travis', [
```

Comment:
We only need a separate `travis` task when tests in travis should differ to those we would want to run locally. So this can just be `'test', 'Run the tests'` and call `grunt test` in the travis script